File: Digits/count_digits02.java
//////////////////////////////////////////////
//
//  File name : count_digits02.java
//  Description : Accept number from user and count there digits
//  Author : Shrishti Sunil Gaikwad
//  Date : 17/05/2025
//
////////////////////////////////////////////////
import java.util.*;

class Digits
{
    public int CountDigits(int iNo)
    {
        int iCnt = 0;
        while(iNo != 0)
        {
            iCnt++;
            iNo = iNo / 10;
        }
        return iCnt;
    }
}

class count_digits02
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number:");
        int iValue = sobj.nextInt();

        Digits dobj = new Digits();

        int iRet = dobj.CountDigits(iValue);

        System.out.println("Number of Digits are :"+iRet);
    }
}

File: Digits/count_digits02.c
///////////////////////////////////////////////////////////////
//
//  File name : count_digits02.c
//  Description : Accept number from user and count there digits
//  Author : Shrishti Sunil Gaikwad
//  Date : 12/05/2025
//
/////////////////////////////////////////////////////////////////

#include <stdio.h>

int CountDigits(int iNo)
{
    int iCnt = 0;

    while(iNo != 0)
    {
        iNo = iNo / 10;
        iCnt++;
    }
    return iCnt;
}

int main()
{
    int iValue = 0,iRet = 0;

    printf("Enter number : \n");
    scanf("%d",&iValue);

    iRet = CountDigits(iValue);

    printf("Number of Digits are : %d\n",iRet);

    return 0;
}